### ISSUE TITLE
Consider using network instead of files

### ISSUE THREAD

**Issue by JohnStrunk** [JohnStrunk/wahoo-results#6]
The scoreboard currently just reads the .do4 result files.
This issue tracks investigating the Dolphin network protocol as a source of data.

Would this allow live race information such as elapsed time during the race or the current heat in the water as opposed to the one that just finished?

**Comment by JohnStrunk** [comment]
After investigating the network protocol a bit, no additional information seems to be available over what can be pulled out of the `*.do4` files. In fact, there seems to be less.

Example network session:
```
autoSendUpdates,ON               # Sent to dolphin
autoSendUpdates set to ON        # Reply
Event 1  Heat 4  Race 4          # After a race, when reset is triggered
1,,False,False
2,22.13,False,False
3,,False,False
4,,False,False
5,,False,False
6,,False,False
7,,False,False
8,,False,False
9,,False,False
10,,False,False
CurrentEventAndHeat,5.0.13.0,2,102,1,5
```

* Only the final time from each lane is sent, making it impossible to check for consistency of the timing result.
* The event description isn't sent automatically, but can be probed
* The event # in the result is the event *index*, not the event number
* `CurrentEventAndHeat` that is sent at the end is for the next race

To summarize:
✔️ Scoreboard wouldn't need access to the data directory
✔️ Scoreboard could automatically load the event & heat info into Dolphin
❌ We can't check for consistency of the race results
❌ Additional logic would be required to convert the event index to the event number
❌ We still don't get "live" race data

**I see no reason to implement this for now, but I'm not against having the capability if someone else wants to work on it**
